### ISSUE TITLE
Retrieve non-follower list

### ISSUE THREAD

**Issue by gkielwasser** [mifi/SimpleInstaBot#138]
Hi,

I would like to extract and retrieve non-followers. Then I would like to use that list to apply the special mode "follow list".

I have seen there is a special mode to unfollow non-mutual but it seems not to be for that purpose.

Thanks for feedback.

**Comment by mifi** [comment]
what do you mean by non-followers? you want to retrieve a list of people who you follow but don't follow you back?

**Comment by gkielwasser** [comment]
Yep. Or at least if not possible, I would like to be able to retrieve a distinct list for followers and another for following.
